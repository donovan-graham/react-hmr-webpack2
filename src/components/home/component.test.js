import React from 'react';
import Home from './component';
import renderer from 'react-test-renderer';

const noOp = () => null;

describe('Home component', () => {
  it('couner at 1', () => {
    const tree = renderer.create(<Home counter={1} onUp={noOp} onDown={noOp} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('couner at 5', () => {
    const tree = renderer.create(<Home counter={5} onUp={noOp} onDown={noOp} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
