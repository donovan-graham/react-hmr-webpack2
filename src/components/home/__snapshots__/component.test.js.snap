// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component couner at 1 1`] = `
<div>
  <h1
    className="fJDhHD"
  >
    Home
  </h1>
  <div>
    Counter: 
    1
  </div>
  <button
    className="eIazjU"
    onClick={[Function]}
  >
    Up
  </button>
  <button
    className="exZpUW"
    onClick={[Function]}
  >
    Down
  </button>
</div>
`;

exports[`Home component couner at 5 1`] = `
<div>
  <h1
    className="fJDhHD"
  >
    Home
  </h1>
  <div>
    Counter: 
    5
  </div>
  <button
    className="eIazjU"
    onClick={[Function]}
  >
    Up
  </button>
  <button
    className="exZpUW"
    onClick={[Function]}
  >
    Down
  </button>
</div>
`;
